File: 프로그래머스/lv1/86051. 없는 숫자 더하기/없는 숫자 더하기.js
function solution(numbers) {
    var answer = 0;
    let x = 0;
    numbers.sort((a, b) => a - b);
    
    for(let i = 0; i <= 9; i++){
        if(numbers[x] !== i){
            answer += i; 
            x--;
        }
        x++;
    }
    return answer;
}

File: Java/프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        boolean list[] = new boolean[10];
        for(int i = 0; i < numbers.length; i++) {
            list[numbers[i]] = true;
        }
        
        for(int i = 0; i < list.length; i++) {
            if(!list[i]) answer += i;
        }
        return answer;
    }
}